diff --git a/node_modules/fluent-ffmpeg/lib/processor.js b/node_modules/fluent-ffmpeg/lib/processor.js
index 36d980a..70a4f03 100644
--- a/node_modules/fluent-ffmpeg/lib/processor.js
+++ b/node_modules/fluent-ffmpeg/lib/processor.js
@@ -488,7 +488,7 @@ module.exports = function(proto) {
               setTimeout(function() {
                 emitEnd(new Error('Output stream closed'));
                 ffmpegProc.kill();
-              }, 20);
+              }, 500);
             });
 
             outputStream.target.on('error', function(err) {
